src/Controller/Api/GalleryController.php

```diff
@@ -198,13 +240,18 @@ public function getGalleryGalleryhasmediasAction($id)
     /**
      * Adds a gallery.
      *
-     * @ApiDoc(
-     *  input={"class"="sonata_media_api_form_gallery", "name"="", "groups"={"sonata_api_write"}},
```

Comment:
Seems that the input parameters and its serialization settings are not present in the upgraded definition.

Reply:
This comment is not addressed yet.